src/Core/src/Platform/Windows/MauiWebView.cs

```diff
@@ -111,9 +117,12 @@ public async void LoadHtml(string? html, string? baseUrl)
 
 		public async void LoadUrl(string? url)
 		{
+			_navigatingToHtmlWithBaseUrl = false;
+
 			Uri uri = new Uri(url ?? string.Empty, UriKind.RelativeOrAbsolute);
 
-			if (!uri.IsAbsoluteUri)
+			if (!uri.IsAbsoluteUri ||
+				uri.AbsoluteUri.ToLowerInvariant().StartsWith(LocalScheme.TrimEnd('/').ToLowerInvariant()) == true)
```

Comment:
Probably more idiomatic to call `string.Equals(x, y, StringComparison.OrdinalIgnoreCase)`

Reply:
This is just a copy of the code in line 146, so do you want me to also change the Starts with() call, potentially introducing a bug?

Reply:
I added two more commits, one where I moved this long line to a new static method. I couldn't find a way to use `string.Equals()`, since the long expression has a `StartsWith()`, not a string comparison. Do you think it's worthwile to call `Uri.TryParse()` and work with the `Uri` object instead?